assembly/compliance-tool/src/test/java/org/eclipse/sw360/antenna/frontend/compliancetool/sw360/reporter/IRGetClearedReleasesTest.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) Bosch.IO GmbH 2020.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.sw360.antenna.frontend.compliancetool.sw360.reporter;
+
+import org.eclipse.sw360.antenna.frontend.compliancetool.sw360.SW360TestUtils;
+import org.eclipse.sw360.antenna.sw360.client.adapter.SW360ComponentClientAdapter;
+import org.eclipse.sw360.antenna.sw360.client.adapter.SW360Connection;
+import org.eclipse.sw360.antenna.sw360.client.adapter.SW360ReleaseClientAdapter;
+import org.eclipse.sw360.antenna.sw360.client.rest.resource.LinkObjects;
+import org.eclipse.sw360.antenna.sw360.client.rest.resource.Self;
+import org.eclipse.sw360.antenna.sw360.client.rest.resource.components.SW360Component;
+import org.eclipse.sw360.antenna.sw360.client.rest.resource.components.SW360ComponentEmbedded;
+import org.eclipse.sw360.antenna.sw360.client.rest.resource.components.SW360SparseComponent;
+import org.eclipse.sw360.antenna.sw360.client.rest.resource.releases.SW360ClearingState;
+import org.eclipse.sw360.antenna.sw360.client.rest.resource.releases.SW360Release;
+import org.eclipse.sw360.antenna.sw360.client.rest.resource.releases.SW360SparseRelease;
+import org.junit.Test;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class IRGetClearedReleasesTest {
+
+    @Test
+    public void executeGetClearedReleasesTest() {
+        SW360Release release = SW360TestUtils.mkSW360Release("test");
+        release.setClearingState("PROJECT_APPROVED");
+        release.setSw360ClearingState(SW360ClearingState.REPORT_AVAILABLE);
+        SW360Connection connection = getSW360Connection(release);
+
+        final IRGetClearedReleases getClearedReleases = new IRGetClearedReleases();
+        final Collection<SW360Release> clearedReleases = getClearedReleases.execute(connection);
+
+        assertThat(clearedReleases).containsExactly(release);
+    }
+
+    /**
+     * create a connection with all mock objects that get
+     * called when executing getting the releases.
+     *
+     * @return mocked sw360 connection
+     */
+    static SW360Connection getSW360Connection(SW360Release release) {
```

Comment:
Maybe a candidate to be refactored into a test helper class, as the method is also used by the test class for not cleared releases.